templates/wizards/files/acn/template.acn

```diff
@@ -0,0 +1,3 @@
+%{JS: Util.fileName('%{TargetPath}')} DEFINITIONS ::= BEGIN
```

Comment:
File: 

```
test DEFINITIONS ::= BEGIN
END
```
when compiled with:

`/opt/asn1scc/bin/asn1.exe test.asn`

gives:

`File temp.asn, line 1:0 required (...)+ loop did not match anything at input 'test'`

while file:

```
Test DEFINITIONS ::= BEGIN
END
```
compiled the same way gives no error, which proves that first capital letter is needed.